RELEASE.md

```diff
@@ -0,0 +1,5 @@
+Release type: patch
```

Comment:
Oh, there's no autopub integration in here T_T

But I'll not reject the PR because of that. I can remove it next when doing the release